common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -801,7 +802,7 @@ def _get_answer_notification(self, render_notifications):
                         progress.frac()[1]
                     ).format(progress=str(progress))
                 else:
-                    answer_notification_message = _('Incorrect')
+                    answer_notification_message = gettext(_('Incorrect'))
```

Comment:
@OmarEdraak , `_(...)` is a dummy and exists just to allow collecting translation strings
https://github.com/Edraak/edraak-platform/blob/b8b86cc8792483b80ce6f42fd73518c4b407cba6/common/lib/xmodule/xmodule/capa_base.py#L42

Therefore, it will not perform a re-translation. Please correct me if I am wrong.

Reply:
That's right shadi. Therefore you should only use the following `gettext('Incorrect')`. Such expression will help in both string collection to pofiles/Transifex (like the dummy function) and do the actual translation (better than the dummy function).

Reply:
Thank you @OmarEdraak . Corrected as requested